draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -246,76 +312,106 @@ Internet-Draft      Network Slice Topology Data Model         March 2023
 
 2.2.  ACTN for Network Slicing
 
-   Since ACTN topology data models are based on the network topology
-   model defined in [RFC8345], the augmentations defined in this
-   document are effective augmentations to the ACTN topology data
-   models, resulting in making the ACTN framework [RFC8453] and data
-   models [I-D.ietf-teas-actn-yang] capable of slicing networks with the
-   required network characteristics.
+   Since ACTN topology data models defined in [RFC8795] are based on the
+   network topology model defined in [RFC8345], the augmentations
+   defined in this document are effective augmentations to the ACTN
+   topology data models, resulting in making the ACTN framework
+   [RFC8453] and data models [I-D.ietf-teas-actn-yang] capable of
+   slicing networks with the required network characteristics.
+
+   The ACTN topology [RFC8795] and tunnel [I-D.ietf-teas-yang-te] data
```

Comment:
RFC 8785 is in principle the TE extension to RFC 8345.
I would avoid to use the term ACTN topology

Reply:
Updated text:
~~~
   The Traffic-Engineering (TE) topology {{!RFC8795}} and tunnel {{?I-D.ietf-teas-yang-te}} data models
   can also be considered as viable models for network slice realization. In this
   case the ACTN data models are used at the MDSC-to-PNC interface (MPI), 
   at the southbound of the NSC, according to {{!RFC8453}} and 
   {{?I-D.ietf-teas-applicability-actn-slicing}}.

~~~